### DISCUSSION TITLE
How does the python scripting widget work with multiple data inputs?

### DISCUSSION THREAD

**Discussion by Bigfoot-solutions** [biolab/orange3#6327] [Q&A]
I am struggling to find any documentation that explains how to use more than one data table within the python scripting widget. I discovered (through trial and error!) that "in_datas" is the plural of "in_data", and I can see both my input tables within in_datas, but I can't figure out how to reference the domains of each of the input tables. 

How can I reference the domain of each input table, like I would with a single input data table using in_data.domain?

**Comment by markotoplak** [comment]
Try `in_datas[0].domain`. Every `Table` has a `.domain` so as long as you can reference the table...

Thanks for reporting that this docs are missing. We used to have inputs written on top-left of the widget, but now I am not sure where that went.
